src/main/java/com/soboleoko/medicalclinic/controller/PatientController.java

```diff
@@ -15,7 +16,7 @@
 public class PatientController {
 
     private final PatientService patientService;
-    private final PatientMapper patientMapper;
+    private final MedicalClinicMapper patientMapper;
```

Comment:
To jest PatientMapper a nie MedicalClinicMapper -> Mappery definiujemy pod konkretne logiczne byty np:
PatientMapper
DoctorMapper

Nie robimy natomiast oddzielnie DoctorMapper oddzielnie DoctorDtoMapper  bo logicznie to ten sam byt